docker-compose.yml

```diff
@@ -47,7 +47,7 @@ services:
   # https://hub.docker.com/r/mozilla/antenna/
   # socorro collector collecting to a separate S3 bucket
   antenna:
-    image: mozilla/antenna:latest
+    image: mozilla/socorro_collector:latest
```

Comment:
We switched the image that we push to during Socorro deploys, so this switches to the current one.